cpp/include/cuml/common/callback.hpp

```diff
@@ -19,7 +19,6 @@
 #include <type_traits>
 
 namespace ML {
-namespace Internals {
```

Comment:
One question: could we keep the internals namespace and update the calls to the callback methods in UMAP?